Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -122,7 +139,6 @@ For a zoomed image try
 
 ```
 
-
 ## Completed Code
 
 We may use the `step-005` branch of the
```

Comment:
We can use the `step-005` branch of the
[github.com/kotlin-hands-on/intro-mpp](https://github.com/kotlin-hands-on/intro-mpp)
repository as the solution for the tasks that we have done above. 
We can also download the
[archive](https://github.com/kotlin-hands-on/intro-mpp/archive/step-005.zip)
from GitHub directly.